stories/IdeaDialog.stories.tsx

```diff
@@ -14,6 +14,7 @@ export default meta;
 type Story = StoryObj<typeof IdeaDialog>;
 
 export const Primary: Story = {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
     render: ({ children, onCancel, ...otherArgs }) => {
```

Comment:
```suggestion
    render: ({ children, ...otherArgs }) => {
```
同时下面改为：
```tsx
                <IdeaDialog
                    centered
                    {...otherArgs}
                    show={showDialog}
                    onCancel={() => setShowDialog(false)}
                >
                    {children}
                </IdeaDialog>
```
这样既不影响现有代码执行，也可让用户覆盖其它属性。